fizzbuzz.py

```diff
@@ -0,0 +1,5 @@
+def fizzbuzz(n):
```

Comment:
ตรงนี้ใช้แบบนี้เลยดีกว่า ย่อแล้วต้องเดา

```suggestion
def fizzbuzz(number):
```